netci.groovy

```diff
@@ -63,23 +63,40 @@ if (branchName.startsWith("features/")) {
 commitPullList.each { isPr ->
   ['debug', 'release'].each { configuration ->
         ['unit32', 'unit64'].each { buildTarget ->
-      def jobName = Utilities.getFullJobName(projectName, "windows_${configuration}_${buildTarget}", isPr)
-            def myJob = job(jobName) {
-        description("Windows ${configuration} tests on ${buildTarget}")
+        def jobName = Utilities.getFullJobName(projectName, "windows_${configuration}_${buildTarget}", isPr)
+        def myJob = job(jobName) {
+            description("Windows ${configuration} tests on ${buildTarget}")
                   steps {
                     batchFile(""".\\build\\scripts\\cibuild.cmd ${(configuration == 'debug') ? '-debug' : '-release'} ${(buildTarget == 'unit32') ? '-test32' : '-test64'} -testDesktop""")
                   }
-                }
+        }
 
-      def triggerPhraseOnly = false
-      def triggerPhraseExtra = ""
-      Utilities.setMachineAffinity(myJob, 'Windows_NT', windowsUnitTestMachine)
-      Utilities.addXUnitDotNETResults(myJob, '**/xUnitResults/*.xml')
-      addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
+        def triggerPhraseOnly = false
+        def triggerPhraseExtra = ""
+        Utilities.setMachineAffinity(myJob, 'Windows_NT', windowsUnitTestMachine)
+        Utilities.addXUnitDotNETResults(myJob, '**/xUnitResults/*.xml')
+        addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
     }
   }
 }
 
+// Windows ES image
+commitPullList.each { isPr ->
+  def jobName = Utilities.getFullJobName(projectName, "windows_debug_es_unit32", isPr)
```

Comment:
Might want to spell out "Spanish" here; es could mean any number of things if you don't already know what this is.

Reply:
ES is the actual language code to my understanding. I was trying to be most specific. Switched to Spanish.

Reply:
It's 100% precise, and 100% obtuse. :smile: